packages/ERTP/src/types-ambient.js

```diff
@@ -217,6 +219,23 @@
 
 // /////////////////////////// Purse / Payment /////////////////////////////////
 
+/**
+ * Issuers first became durable with mandatory recovery sets. Later they were
+ * made optional, but there is no support for converting from one state to the
+ * other. Thus, absence of a `RecoverySetsOption` state is equivalent to
+ * `'hasRecoverySets'`. In the absence of a `recoverySetsOption` parameter,
+ * upgradeIssuerKit defaults to the predecessor's `RecoverySetsOption` state, or
+ * `'hasRecoverySets'` if none.
+ *
+ * At this time, a `'noRecoverySets'` predecessor cannot be upgraded to a
+ * `'hasRecoverySets'` successor. If it turns out this transition is needed, it
+ * can likely be supported in a future upgrade.
```

Comment:
Please also state the restriction in the opposite direction, using the same "At this time..." wording.

Reply:
done.